task-list/app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,20 @@
 <header>
   🔅🔅🔆 Task Organizer 🔆🔅🔅
 </header>
+<nav>
```

Comment:
Nice use of a navigation bar now that the site has enough pages now to warrant it.